projects/aws/image-builder/builder/utils.go

```diff
@@ -14,9 +15,49 @@ import (
 	"time"
 
 	releasev1 "github.com/aws/eks-anywhere/release/api/v1alpha1"
-	"sigs.k8s.io/yaml"
```

Comment:
Can we continue to use `sigs.k8s.io/yaml` for the EKS-A release and bundle manifest unmarshalling since those are Kubernetes CRDs, and use the `github.com/ghodss/yaml` for the EKS-D releases file?